src/tutorial/TutorialPhase.cs

```diff
@@ -22,12 +22,6 @@ public abstract class TutorialPhase
     [JsonProperty]
     public bool HandlesEvents { get; protected set; } = true;
 
-    /// <summary>
-    ///   True when this tutorial uses an exclusive popup preventing all other GUI from being interacted with
-    /// </summary>
-    [JsonProperty]
-    public bool Exclusive { get; protected set; }
```

Comment:
Related to my earlier comment, I guess removing this makes sense of the dialog type already handles being exclusive and reporting that...

Reply:
Yeah, I think the exclusive check in GUICommon is already inclusive.